static/app/views/settings/projectIssueGrouping/upgradeGrouping.tsx

```diff
@@ -57,6 +57,7 @@ function UpgradeGrouping({
       return;
     }
     handleOpenConfirmModal();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
you would have to update many things in this file to make this warning go away, so I think it's fine to have this comment for now. We can revisit this afterwards